setup.cfg

```diff
@@ -29,14 +29,17 @@ classifiers =
 python_requires = >=3.6
 packages = find:
 include_package_data = True
-setup_requires = setuptools_scm
+setup_requires =
+  setuptools_scm
+  importlib_resources;python_version<"3.7"
```

Comment:
do we need this here?

Reply:
Probably not, I was wondering if I added it there could I remove the check for `pkg_resources` and the answer was a no. Which I find odd since surely it should be installed at setup time like `scm` is.

Reply:
Where is this version calc even called during setup time? I would have expected scm to just handle it. Or do we import this into setup.py in some way?

Reply:
Who knows.